src/jamf_pro_sdk/clients/classic_api.py

```diff
@@ -173,6 +178,69 @@ def delete_computer_by_id(self, computer: ComputerId) -> None:
         computer_id = ClassicApi._parse_id(computer)
         self.api_request(method="delete", resource_path=f"computers/id/{computer_id}")
 
+    def set_computer_unmanaged_by_id(self, computer: ComputerId) -> None:
+        """Sets the management status to `unmanaged` for a single computer using the ID
+
+        ..important::
+            This action does not remove the management framework or mdm profile from the device
+
+        :param computer: A computer ID or supported Classic API model.
+        :type computer: Union[int,  ~jamf_pro_sdk.models.classic.computers.Computer, ComputersItem]
+        """
+        data = {
+            "general": {
+                "remote_management": {
+                    "managed": False,
+                    "management_username": "",
+                    "management_password": "",
+                }
+            }
+        }
+        update_management = ClassicComputer(**data)
+        computer_id = ClassicApi._parse_id(computer)
+        self.api_request(
+            method="put", resource_path=f"computers/id/{computer_id}", data=update_management
+        )
+
+    def set_computer_managed_by_id(
+        self, computer: ComputerId, management_user: str = "admin", management_password: str = None
+    ) -> None:
+        """Sets the management status to `managed` for a single computer using the ID.
+
+        ..important::
```

Comment:
```suggestion
        .. important::
```
Missing a space after `..`.